src/lib/react/ReactMacro.hx

```diff
@@ -295,6 +295,37 @@ class ReactMacro
 		return fields;
 	}
 
+	static function buildJsxStatic():Array<Field> {
```

Comment:
Could use a comment saying "Add a `displayName` to functional components"

Reply:
Sure! :)